appserver/tests/pom.xml

```diff
@@ -42,6 +42,18 @@
 -->
 
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd"> <modelVersion>4.0.0</modelVersion>
+    <dependencies>
+        <dependency>
+            <groupId>jakarta.platform</groupId>
+            <artifactId>jakarta.jakartaee-web-api</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>jakarta.xml.bind</groupId>
+            <artifactId>jakarta.xml.bind-api</artifactId>
```

Comment:
I'm not a fan of this being defined up here.
It's going to end up being pulled into everything up here when not necessarily everything needs it e.g. payara-samples/test-utils, everything under quicklook.

I'd at least move this down to payara-samples, and remove the hardcoded version - the version is already defined in dependency management of [payara-bom](https://github.com/payara/Payara/blob/Payara6/api/payara-bom/pom.xml#L310)

Reply:
Ok, I will move it to payara-samples,
but the version defined in payara-bom is <jaxb-api.version>3.0.1</jaxb-api.version>
Unhappily the test only works with 2.3.3
With 3.0.1 it result in
java.lang.NoClassDefFoundError: javax/xml/bind/annotation/XmlElement

Reply:
That to me stinks of something not being right.
If I'm not mistaken 2.3.3 is the version for Jakarta EE 8, 3.0.1 is the version for Jakarta EE 9 (namespace change)